docs/source/user_guides/superset_guide.rst

```diff
@@ -84,35 +86,102 @@ Using Superset
+aggregates the labels of nodes visited at the end of dialog graph traversal,
+i.e. nodes that terminate the dialog.
+
+`dff-nodes-stats` uses the following variables to store the data:
+
+* The `context_id` field can be used to distinguish dialog contexts from each other and serves as a user identifier.
+* `request_id` is the number of the dialog turn at which the data record was emitted. The data points can be aggregated over this field, showing the distribution of a variable over the dialog history.
+* The `data_key` field contains the name of the extractor function that emitted the given record. Since in most cases you will only need the output of one extractor, you can filter out all the other records using filters.
+* Finally, the `data` field is a set of JSON-encoded key-value pairs. The keys and values differ depending on the extractor function that emitted the data (you can essentially save arbitrary data under arbitrary keys), which makes filtering the data rows by their `data_key` all the more important. The JSON format implies that individual values need to be extracted using the Superset SQL functions (see below).
+
+
+.. code-block::
+
+    JSON_VALUE(data, '$.key')
+    JSON_VALUE(data, '$.outer_key.nested_key')
+
+**Chart creation**
+
+.. note::
+
+    Chart creation is described in detail in the official Superset documentation.
+    We suggest that you consult it in addition to this section:
+    `link <https://superset.apache.org/docs/creating-charts-dashboards/exploring-data/#pivot-table>`_.
+
+Creating your own chart is as easy as navigating to the `Charts` section of the Superset app
+and pressing the `Create` button.
+
+Initially, you will be prompted for the dataset that you want to use as well as for the chart type.
+The Superset GUI provides comprehensive previews of each chart type making it very easy
+to find the exact kind that you need.
+
+At the next step, you will be redirected to the chart creation interface.
+Depending on the kind of chat that you have chosen previously, menus will be available
+to choose a column for the x-axis and, optionally, a column for the y-axis. As mentioned above,
+a separate menu for data filters will also be available. If you need to use the data
+from the `data` column, you will need to find the `custom_sql` option when adding the column
+and put in the extraction expression, as shown in the examples above.
+
+**Persisting the chart configuration**
+
+If you define your own charts, it's important to save their configuration to the file system of your
+host machine, so that no information is lost when the Superset container or the Docker process is restarted.
+
+The most convenient way to do that is to export the dashboard configuration as a whole. Navigate to the
+`Dashboards` section of the Superset application, locate your dashboard (named `DFF statistics` per default).
+Then press the `export` button on the right and save the zip file to any convenient location.
```

Comment:
I don't think that that is the most convenient way.

Superset doc also recommends using a separate container for configuration:
![image](https://github.com/deeppavlov/dialog_flow_framework/assets/61429541/8ee16769-95b2-4b49-981b-f15b38514a65)
https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/#3-launch-superset-through-docker-compose

And they have an example of a managed database here:
https://github.com/apache/superset/blob/7317d9c0b2f0782b161a19c9faf95fd4d8634619/docker-compose-non-dev.yml#L35-L42

Reply:
Added a configuration file to override the standard config